src/components/common/dialogs/DeleteDialog.tsx

```diff
@@ -23,7 +23,7 @@ export const DeleteDialog: React.FC<DeleteDialogProps> & { Description?: React.F
         <ConfirmationDialog.ButtonGroup>
             <div className="flex right">
                 <button type="button" className="cta cancel cta-cd-delete-modal ml-16" onClick={props.closeDelete}>Cancel</button>
-                <button type="button" className="cta delete ml-16" onClick={props.delete}>{props.deletePrefix}Delete</button>
+                <button type="button" className="cta delete cta-cd-delete-modal ml-16" onClick={props.delete}>{props.deletePrefix}Delete</button>
```

Comment:
I assume this change is intentional (As this code is generic, not specific to this particular story )

Reply:
Added intentionally as the width of the button was deferring from demo1 version & `cancel` button itself.